.github/workflows/build_and_test.yaml

```
@@ -1,32 +1,33 @@
 name: Build and test libcbv2g
 on: 
   pull_request: {}
+  workflow_dispatch:
+    inputs:
+      runner:
+        description: Which runner to use
+        type: choice
+        default: 'ubuntu-22.04'
+        required: true
+        options:
+          - 'ubuntu-22.04'
+          - 'large-ubuntu-22.04-xxl'
+  schedule:
+    - cron: '37 13,1 * * *'
 jobs:
-  build_and_test:
-    name: Build and test
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout libcbv2g
-        uses: actions/checkout@v3
-        with:
-          path: source
-      - name: Setup run scripts
-        run: |
-          mkdir scripts
-          rsync -a source/.ci/build-kit/ scripts
-      - name: Pull docker container
-        run: |
-          docker pull --quiet ghcr.io/everest/build-kit-alpine:v1.2.0
-          docker image tag ghcr.io/everest/build-kit-alpine:v1.2.0 build-kit
-      - name: Build and test
-        run: |
-          docker run \
-          --volume "$(pwd):/ext" \
-          --name test-container \
-          build-kit run-script build_and_test
-      - name: Archive test results
-        if: always()
-        uses: actions/upload-artifact@v3
-        with:
-          name: ctest-report
-          path: ${{ github.workspace }}/ctest-report
+  ci:
+    name: Build and Test
+    uses: everest/everest-ci/.github/workflows/continuous_integration.yml@v1.4.6
+    permissions:
+      contents: read
+    secrets:
+      coverage_deploy_token: ${{ secrets.SA_GITHUB_PAT }}
+    with:
+      runner: ${{ inputs.runner || 'ubuntu-22.04' }}
+      artifact_deploy_target_repo: EVerest/everest.github.io
```

```suggestion
      artifact_deploy_target_repo: ""
```

At the moment it wouldn't make a difference settings this to empty string. You could argue this should be optional, but since it is formulated in a more generic way it could be that ctest reports for example are pushed to this repository as well. Which is not the case at the moment, but coul be an extension in the future, theoretically. So in this sense it would make sense to not set it to empty string.. your decision :)

---

Then I'll leave it as it is :+1: